docs/api/migration-guides/qiskit-quantum-instance.mdx

```diff
@@ -144,481 +144,463 @@ primitives **expose a similar setting through their interface**:
 
 ## Code examples
 
-```{eval-rst}
-.. dropdown:: Example 1: Circuit Sampling with Local Simulation
-    :animate: fade-in-slide-down
+### Example 1: Circuit Sampling with Local Simulation
```

Comment:
There are 4 examples that were dropdowns. For each of them, I added a header and dedented all the content. For the content itself, I replaced two backticks with a single backtick. I also converted code blocks to Markdown.